homeassistant/components/mazda/sensor.py

```diff
@@ -41,92 +41,142 @@ class MazdaSensorEntityDescription(SensorEntityDescription):
+    return data["status"]["fuelRemainingPercent"] is not None
+
+
+def _fuel_distance_remaining_supported(data):
+    """Determine if fuel distance remaining is supported."""
+    return data["status"]["fuelDistanceRemainingKm"] is not None
+
+
+def _odometer_supported(data):
+    """Determine if odometer is supported."""
+    return data["status"]["odometerKm"] is not None
+
+
+def _front_left_tire_pressure_supported(data):
+    """Determine if front left tire pressure is supported."""
+    return data["status"]["tirePressure"]["frontLeftTirePressurePsi"] is not None
+
+
+def _front_right_tire_pressure_supported(data):
+    """Determine if front right tire pressure is supported."""
+    return data["status"]["tirePressure"]["frontRightTirePressurePsi"] is not None
+
+
+def _rear_left_tire_pressure_supported(data):
+    """Determine if rear left tire pressure is supported."""
+    return data["status"]["tirePressure"]["rearLeftTirePressurePsi"] is not None
+
+
+def _rear_right_tire_pressure_supported(data):
+    """Determine if rear right tire pressure is supported."""
+    return data["status"]["tirePressure"]["rearRightTirePressurePsi"] is not None
+
+
+def _fuel_remaining_percentage_value(data, unit_system):
+    """Get the fuel remaining percentage value."""
+    return data["status"]["fuelRemainingPercent"]
+
+
+def _fuel_distance_remaining_value(data, unit_system):
+    """Get the fuel distance remaining value."""
+    return round(
+        unit_system.length(data["status"]["fuelDistanceRemainingKm"], LENGTH_KILOMETERS)
+    )
+
+
+def _odometer_value(data, unit_system):
+    """Get the odometer value."""
+    return round(unit_system.length(data["status"]["odometerKm"], LENGTH_KILOMETERS))
+
+
+def _front_left_tire_pressure_value(data, unit_system):
+    """Get the front left tire pressure value."""
+    return round(data["status"]["tirePressure"]["frontLeftTirePressurePsi"])
+
+
+def _front_right_tire_pressure_value(data, unit_system):
+    """Get the front right tire pressure value."""
+    return round(data["status"]["tirePressure"]["frontRightTirePressurePsi"])
+
+
+def _rear_left_tire_pressure_value(data, unit_system):
+    """Get the rear left tire pressure value."""
+    return round(data["status"]["tirePressure"]["rearLeftTirePressurePsi"])
+
+
+def _rear_right_tire_pressure_value(data, unit_system):
+    """Get the rear right tire pressure value."""
+    return round(data["status"]["tirePressure"]["rearRightTirePressurePsi"])
+
+
 SENSOR_ENTITIES = [
     MazdaSensorEntityDescription(
         key="fuel_remaining_percentage",
         name_suffix="Fuel Remaining Percentage",
         icon="mdi:gas-station",
         unit=lambda unit_system: PERCENTAGE,
-        is_supported=lambda data: data["status"]["fuelRemainingPercent"] is not None,
-        value=lambda data, unit_system: data["status"]["fuelRemainingPercent"],
+        is_supported=_fuel_remaining_percentage_supported,
```

Comment:
These lambdas didn't span more than one line. Why did we replace them?

Reply:
For consistency. I thought it looked a bit strange only replacing some of them. I'll change them back.

Reply:
@MartinHjelmare I changed the ones that were one line back into lambdas.